docs/layers/grid-layer.md

```diff
@@ -62,6 +62,23 @@ Color scale domain, default is set to the range of point counts in each cell.
 Color ranges as an array of colors formatted as `[255, 255, 255]`. Default is
 [colorbrewer](http://colorbrewer2.org/#type=sequential&scheme=YlOrRd&n=6) `6-class YlOrRd`.
 
+##### `getColorValue` (Function, optional)
+
+- Default: `points => points.length`
+
+`getColorValue` is the accessor function to get the value that cell color is based on. 
+It takes an array of points inside each cell as arguments, returns a value. For example, 
+You can pass in `getColorValue` to color the cells by avg/mean/max of a specific attributes of each point.
+By default `getColorValue` returns the length of the points array.
+
+##### `coverage` (Number, optional)
+
+- Default: `1`
+
+Cell size multiplier, clamped between 0 - 1. The final size of cell
```

Comment:
Specify whether coverage is linear or area based? I assume this is just a clamped `radiusScale` factor, i.e. linear.